requirements.md

```diff
@@ -66,10 +66,65 @@ These are mostly in response to trends we see in issues made
 - Switch between slower HiFi resampler and a fast LoFi one.
 
 ### Needed by Bevy
-Bevy is a widely used game engine build around the ECS model. Because bevy uses ECS it has specific demands from an audio engine.
 
-*TODO*
+Bevy is a popular game engine built around the ECS paradigm.
+Its ECS is highly flexible, so there are many valid ways to integrate
+audio engines. However, idiomatic integrations
+should strive to satisfy a few conditions:
 
+1. ECS-driven design.
+2. Broad platform support.
+3. Integration with Bevy's asset system.
+
+Bevy is looking for a _first-class_ audio solution.
+Playing samples, applying effects, routing audio, and
+even authoring effects should feel natural, not tacked-on.
+Adhering to these conditions will help guide engines and integrations
+towards that first-class experience.
+
+#### ECS-driven design
+
+Increasingly, crate authors within Bevy's ecosystem are choosing to integrate
+deeply with the ECS; they are choosing _ECS-driven_ design.
+This provides quite a few advantages to both crate authors and users.
+For example, keeping track of emitter positions for spatial audio is easy
+when emitter and transform components share the same entity.
```

Comment:
with transform components you mean effects like `low_pass`?

Reply:
No, Bevy's `GlobalTransform`.

Reply:
Suggestion: 

- replace 'transform' with a link to bevy's [GlobalTransform](https://docs.rs/bevy/latest/bevy/prelude/struct.GlobalTransform.html). 
- replace emitter with 'sound emitter' (though that's clear if you keep reading it lowers cognitive load to have the word sound here).